src/commands/basemaps-github/create-pr.ts

```diff
@@ -104,18 +110,18 @@ export const CommandCreatePRArgs = {
     defaultValue: () => 'linz/basemaps-config',
     defaultValueIsSerializable: true,
   }),
+  configType: option({
+    type: optional(oneOf(Object.values(ConfigType))),
+    long: 'config-type',
+    description: `Basemaps config file type, includes ${[...Object.values(Category)].join(', ')}`,
```

Comment:
Validating against ConfigType but docs are using Category, what is the difference?

Reply:
Very good pick. I just copied from other one and forgot to update the type. Just fixed.